flink-streaming-java/src/test/java/org/apache/flink/streaming/api/graph/StreamGraphGeneratorTest.java

```diff
@@ -566,15 +567,15 @@ public void testMaxParallelismWithConnectedKeyedStream() {
         int maxParallelism = 42;
 
         StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
-        DataStream<Integer> input1 = env.fromElements(1, 2, 3, 4).setMaxParallelism(128);
-        DataStream<Integer> input2 = env.fromElements(1, 2, 3, 4).setMaxParallelism(129);
+        DataStream<Long> input1 = env.fromSequence(1, 4).setMaxParallelism(128);
+        DataStream<Long> input2 = env.fromSequence(1, 4).setMaxParallelism(129);
```

Comment:
Is it just for test coverage?

Reply:
Yes, this fixes test failures that arise because of this: https://github.com/apache/flink/pull/23553/files#diff-4a5eb9032bed78bb9f18e6523d4f7b3dc86ed10e3a3689757c1c4fa2335e7255R1307
the SingleOutputStreamOperator caps max parallelism to 1. 
Since this PR is already pretty sizable, it seemed appropriate to postpone dealing with this when we work on `fromSequence`.